MoltenVK/MoltenVK/Commands/MVKMTLBufferAllocation.h

```diff
@@ -101,8 +103,9 @@ class MVKMTLBufferAllocationPool : public MVKObjectPool<MVKMTLBufferAllocation>,
     NSUInteger _allocationLength;
     NSUInteger _mtlBufferLength;
     MTLStorageMode _mtlStorageMode;
-	MVKSmallVector<id<MTLBuffer>, 64> _mtlBuffers;
 	bool _isThreadSafe;
```

Comment:
Prefer to move `_isThreadSafe` to below `_lockedBuffers` to avoid gaps in the struct.